src/routes/Home.js

```diff
@@ -0,0 +1,62 @@
+import MemberList from '../components/MemberList';
+import { Component } from '../core/component';
+import { navigate, routeRender } from '../core/router';
+import {
+  deleteData,
+  memberStore,
+  getMembersData,
+  searchData,
+} from '../store/memberStore';
+
+export default class Home extends Component {
+  async render() {
+    this.el.innerHTML = `<header class="header">
+    <div class="title">직원 관리 시스템</div>
+    <div class="container">
+      <input class="search" placeholder="이름으로 검색해주세요"/>
+      <button class="button-home" id="navigate-write">등록</button>
```

Comment:
```suggestion
<button class="button-home" id="navigate-write" onClick="${() =>
        navigate("/#/write")}">등록</button>
```

이렇게 직접 inline 으로 적어줘도 되는 것 같은데 어떠신가요??

Reply:
innerHTML 내부에 작성한 인라인 스크립트가 정상적으로 작동하지 않아서 addEventListener을 사용했습니다!

저는 XSS 보안때문에 innerHTML 내부 인라인 스크립트에서 외부 함수를 호출할 수 없는 걸로 알고 있는데
혹시 제가 잘못 알고 있는 걸까요!?

Reply:
엇 인라인으로 호출하면 실행이 안 되나요?? 제가 테스트해보고 리뷰 남겼었던 걸로 기억하는데, XSS 이슈를 생각하면 사실 addEventListener 로도 신경써야 하는 부분들이 많아서, react에서 함수를 전달하는 방식으로 제가 리뷰를 남겼던 것 같습니다. 
가현님이 잘못 알고 있다기보다는, 제가 다른 방향으로 리뷰를 남겼던 것 같아요!